azure-pipelines-official.yml

```diff
@@ -317,8 +317,9 @@ stages:
         command: push
         packagesToPush: '$(Build.SourcesDirectory)\artifacts\LanguageServer\*.nupkg'
         allowPackageConflicts: false
+        feedsToUse: select
```

Comment:
This is documented as the default?

Reply:
yeah I don't need specify this value.  However CI passed on this commit and I have another PR out to delete this entire step (as soon as dnceng helps me fix signing) - https://github.com/dotnet/roslyn/pull/68504

So going to leave this as-is for expediency